neurom/core/_neuron.py

```diff
@@ -152,6 +154,51 @@ def graft_neuron(root_section):
     return Neuron(soma=Soma(root_section.points[:1]), neurites=[Neurite(root_section)])
 
 
+def iter_positions(morph, neurite_filter, sample_distance):
+    """
+    Iterator for positions in space of points every <sample_distance> um
+
+    Args:
+        morph (neurom.FstNeuron): morphology
+        neurite_type: (neurom.NeuriteType): filter by this type of neurite
```

Comment:
neurite_type is not a neurom.NeuriteType, but a filter function on neurite types (as the other functions in that file)